Formula/azure-cli@1.0.rb

```diff
@@ -0,0 +1,36 @@
+require "language/node"
+
+class AzureCliAT10 < Formula
+  desc "Official Azure CLI"
+  homepage "https://github.com/azure/azure-xplat-cli"
+  url "https://github.com/Azure/azure-xplat-cli/archive/v0.10.12-May2017.tar.gz"
+  version "1.0.10.13"
+  sha256 "60194e770b8dca0485db9d4c99f9cd432f7b43096cc5ad0353f52fd5b1b29181"
+  head "https://github.com/azure/azure-xplat-cli.git", :branch => "dev"
+
+  bottle do
+    cellar :any_skip_relocation
+    sha256 "0849685ed65f1bfa991c3ffbfaa281f9051d33b626b79b12b1f5341f5925976d" => :sierra
+    sha256 "6bcf4ea33dad69d8c703724e09468e08c98d053456fb41a94f6b5877d9a8c243" => :el_capitan
+    sha256 "16184e20ae089b157c2e513c1b194ace66d1f46da5eb2869fb9261b1f71aaf1f" => :yosemite
+  end
+
+  depends_on "node"
+  depends_on :python => :build
```

Comment:
I just renamed that Formula, I think changes to the old azure-cli formula should be part of another PR: